### PR TITLE
Add rlobillo as QE cherry-pick approver for OCP on OSP repos

### PR DESCRIPTION
**Pull request by rlobillo**
Based on the new [backport process](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit#heading=h.bmt7mxrm6cr), adding @rlobillo to @eurijon and @itzikb-redhat as the QE cherry-pick approvers for OCP on OSP team repositories.

### CODE REVIEW

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Comment by mandre** [comment]
We need your approval to get Ramón in the QE cherry-pick approver list for repos you maintain. Could you take a look please?
/assign annazivkovic dougbtv gryf yuqi-zhang

**Comment by mandre** [comment]
@yuqi-zhang, exactly, for MCO this would be for patches affecting the OpenStack templates. Ramón is QE for OpenShift on OpenStack and we want to grant him permission as Jon's availability is going to be spotty over the next few months. Jon is another OpenShift on OpenStack QE who already has the permission to set the cherry-pick-approved label.

CNO and installer also have some openstack specific code.

**Review by yuqi-zhang — APPROVED** [review]
Makes sense, thanks!

**Comment by yuqi-zhang** [comment]
Hmm, did I have to explicitly

/approve

**Review by gryf — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
